recipes/vlc/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+
+
+class VlcConan(ConanFile):
+    name = "vlc"
+    license = "GPL"
+    homepage = "https://www.videolan.org/vlc/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "VLC is a free and open source cross-platform multimedia " \
+                  "player and framework that plays most multimedia files as " \
+                  "well as DVDs, Audio CDs, VCDs, and various streaming protocols."
+    topics = ("vlc", "multimedia", "sound")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "pkg_config"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_lua": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_lua": False,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def system_requirements(self):
+        packages = []
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            if tools.os_info.with_apt:
+                packages = ["libxcb-composite0-dev", "libxcb-xv0-dev"]
```

Comment:
```suggestion
                packages = ["libxcb-composite0-dev", "libxcb-xv0-dev"]
```

Forbidden. You have to create a new conan package for composite first. Take a look on libusb as example.